datetime_formatter/__openerp__.py

```diff
@@ -0,0 +1,12 @@
+# -*- encoding: utf-8 -*-
+
+{
+    "name": "Date & Time Formatter",
+    "summary": "Helper functions to give correct format to date[time] fields",
+    "version": "1.0.0",
+    "author": "Odoo Community Association (OCA)",
+    "category": "Tools",
+    "depends": [
+        "base",
```

Comment:
Seems a common practice in Odoo source code. It may be redundant, but the developer reading this manifest immediately knows that this module needs nothing but the bare Odoo server. It's an opinion, though.